### PR TITLE
fix: add `startAt` and `endAt` for Tekton runs

### PR DESCRIPTION
**Pull request by lijie0123**
Why:
- currently the pipeline start at and end at are not properly set

### CODE REVIEW

File: tibuild/pkg/webhook/service/cloudevent.go [hunk 1]
```diff
@@ -94,11 +94,13 @@ func toDevbuildPipeline(pipeline tekton.PipelineRun) (*rest.TektonPipeline, erro
...
-		GitHash:      parseGitHash(pipeline),
-		OciArtifacts: convertOciArtifacts(pipeline),
-		Images:       images,
+		Name:            pipeline.Name,
+		Platform:        parsePlatform(pipeline),
+		GitHash:         parseGitHash(pipeline),
+		OciArtifacts:    convertOciArtifacts(pipeline),
+		Images:          images,
+		PipelineStartAt: &pipeline.Status.StartTime.Time,
+		PipelineEndAt:   &pipeline.Status.CompletionTime.Time,
```
Comment (wuhuizuo) [inline tibuild/pkg/webhook/service/cloudevent.go:103]: Why not just name them `StartedAt` and `EndAt`?

Comment (lijie0123) [reply tibuild/pkg/webhook/service/cloudevent.go]: renamed

**Pull request merged**